src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -1,5 +1,6 @@
 import { Box } from '@chakra-ui/react'
 import { cosmosAssetId, fromAssetId, osmosisAssetId } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/investor-foxy/dist/utils'
```

Comment:
This whole commit will effectively get turned into something nice and clean, simply committed it as an intermediary one to track which places used `selectFoxEthLpAccountsOpportunitiesAggregated` and need the opportunitiesSlice flavor instead - see current foxEthSlice implementation here:

https://github.com/shapeshift/web/blob/a716c25bd284c58df1872e7829926f2f60e94955/src/state/slices/foxEthSlice/selectors.ts#L81-L116

Reply:
Note for self: Bring over the `baseFarmingOpportunity` and the other constants from `foxEthCommon` and key them by `LpId | StakingId` instead of a earn making util that would bring more bloat